torch/onnx/symbolic.py

```diff
@@ -355,6 +355,11 @@ def replication_pad(g, input, padding):
 replication_pad3d = replication_pad
 
 
+def upsample_nearest2d(g, input, scale_factor):
+    return g.op("Upsample", input, width_scale_f=scale_factor,
+                height_scale_f=scale_factor, mode_s="nearest")
```

Comment:
Doesn't this also get sizes?

Reply:
Just `scale_factor`. If `sizes` is specified, it's converted to `scale_factor` before calling into ATen.